examples/pytorch/text-generator/README.md

```diff
@@ -4,9 +4,7 @@ _WARNING: you are on the master branch; please refer to examples on the branch c
 
 This example shows how to deploy a realtime text generation API using a GPT-2 model from Hugging Face's transformers library.
 
-<br>
-
-## Implement your predictor
+## Implement your Predictor
 
 1. Create a Python file named `predictor.py`.
```

Comment:
Instead of these 3 numbered bullets, what about having a single sentence like `Create a Python file named predictor.py with the following contents`? I'm thinking this way it's shorter and we get rid of 2 lines. What is your thought on this?